src/engine/input/key_mapping/SpecifiedInputKey.cs

```diff
@@ -59,7 +59,23 @@ public override string ToString()
         {
             // TODO: add translation support for key names like Right, and Left etc.
             // TODO: https://github.com/Revolutionary-Games/Thrive/issues/1895
-            text += ((KeyList)Code).ToString();
+
+            // text += ((KeyList)Code).ToString();
+
+            var keyname = ((KeyList)Code).ToString();
+
+            if (keyname == "Left")
+                text += TranslationServer.Translate("LEFT");
+            else if (keyname == "Right")
+                text += TranslationServer.Translate("RIGHT");
+            else if (keyname == "Up")
+                text += TranslationServer.Translate("UP");
+            else if (keyname == "Down")
+                text += TranslationServer.Translate("DOWN");
+            else if (keyname == "Print")
+                text += TranslationServer.Translate("PRINT");
+            else
+                text += ((KeyList)Code).ToString();
```

Comment:
Maybe this could also use the new C# 8 switch expression used in the else block for cleaner look. So something like:
```
text += keyname switch
{
   "Left" => TranslationServer.Translate("LEFT"),
   "Right" => TranslationServer.Translate("RIGHT"),
   "Up" => TranslationServer.Translate("UP"),
   ...
};
```